.buildkite/pipeline.yml

```diff
@@ -55,6 +55,8 @@ steps:
         command: "make -C /agent generate-docker-images"
         agents:
           image: "docker.elastic.co/ci-agent-images/serverless-docker-builder:0.0.11"
+          cpu: "4"
+          memory: "6G"
```

Comment:
I am not sure if that has the desired effect of being inherited by the steps resulting from this command.